index.html

```diff
@@ -4116,91 +4036,328 @@ <h3>Parsing content</h3>
+          injection, redirect to malicious web page, phishing user location,
+          causing side actions such as installing applications, trigger
+          automated dispatching or other actions.
+        </li>
+        <li>
+          <strong>Malicious MitM user</strong>: any MitM (man in the middle)
+          style attack between Web NFC implementation and an <a>NFC adapter</a>
+          in a user device, including attempts to interact with a web site using
+          Web NFC by presenting modified or replayed NDEF records.
+        </li>
+      </ul>
+    </p>
   </section>
 
-  <section class="informative"> <h3>Security policies</h3>
+  <section class="informative"> <h3>Threats</h3>
     <p>
-      This section summarizes the security policies which are specified as
-      normative requirements in the respective algorithms of this
-      specification.
+      An introduction to NFC security is found <a href="https://resources.infosecinstitute.com/near-field-communication-nfc-technology-vulnerabilities-and-principal-attack-schema/">here</a>.
+      Potential threats for Web NFC are given below.
     </p>
+
+    <section> <h4>Fingerprinting and data collection</h4>
+      <dl>
+        <dt><strong>Threat description</strong></dt>
+        <dd>
+          Malicious web page collects user data, identity or other privacy
+          sensitive attributes (such as location) without user consent and
+          exposes it to third parties (writing it to NFC tags, pushing to NFC
+          peers).
+        </dd>
+        <dt><strong>Affected assets</strong></dt>
+        <dd>
+          User data, user identity or other privacy sensitive attributes
+        </dd>
+        <dt><strong>Actors</strong></dt>
+        <dd>
+          Malicious web page owner using Web NFC, malicious tag owner.
+        </dd>
+        <dt><strong>Mitigation, comments</strong></dt>
+        <dd>
+          The user SHOULD be able to be aware of what data can be shared using
+          NFC from the given web page.
+          Use permissions and user prompts for accessing personal data,
+          minimize user data exposed to NFC.
+          A NFC tag SHOULD NOT ever trigger a user’s device to navigate
+          to a website without asking permission, unless the site has been in
+          the foreground or has been brought to the foreground and has been
+          granted permission. User agents SHOULD take into account the
+          security and privacy measures listed in the
+          <a href="http://dev.w3.org/geo/api/spec-source.html#security">
+          Geolocation API</a>.
+        </dd>
+      </dl>
+    </section>
+
+    <section> <h4>NFC tag modification</h4>
+      <dl>
+        <dt><strong>Threat description</strong></dt>
+        <dd>
+          An NFC tag is being modified without user consent.
+          This might enable further attacks using a malicious tag
+          or can be a Denial of Service attack to make one or more tags
```

Comment:
You write MITM as abbreviation but not DOS which is a much more common abbreviation

Reply:
Actually it's DoS for that one, DOS is the ancient OS fossil :) 
https://en.wikipedia.org/wiki/Denial-of-service_attack

Reply:
Well I used it a lot, so am I ancient too? ;)

Reply:
Me too, I was self-sarcastic :)